algorithms/alg.verify-tx.tex

```diff
@@ -0,0 +1,12 @@
+\begin{algorithm}[H]
+    \caption{\label{alg.verify-tx} A function to verify that a transaction is conforming to the provided event and is included in the provided block.}
+    \begin{algorithmic}[1]
+        \Function{\sf verify-tx$_e$}{$\alpha$}
```

Comment:
I'd prefer to see the arguments passed into this function literally: b, tx, addr, amount, instead of a/e.